datumaro/components/crypter.py

```diff
@@ -7,19 +7,29 @@
 
 from cryptography.fernet import Fernet, InvalidToken
 
+from datumaro.components.errors import DatumaroError
+
 
 class Crypter:
-    FERNET_KEY_LEN = 44
+    # Prefix (datum-) = 6 and Fernet = 44, 6 + 44 = 50
+    FERNET_KEY_LEN = 50
+    KEY_PREFIX = b"datum-"
+    KEY_PREFIX_LEN = len(KEY_PREFIX)
 
     def __init__(self, key: Union[str, bytes]) -> None:
         if isinstance(key, str):
             key = key.encode()
-        self._key = key
+        if len(key) != self.FERNET_KEY_LEN:
+            raise DatumaroError(
+                f"Key length should be {self.FERNET_KEY_LEN}, "
+                f"but your key length is {len(key)} (key={key})."
+            )
+        self._key = key[self.KEY_PREFIX_LEN :]
```

Comment:
maybe `key[self.KEY_PREFIX_LEN:]`?

Reply:
I did `black` for code formatting and it made as is.